.github/workflows/test_deploy.yml

```
@@ -30,7 +31,7 @@ jobs:
     - uses: rokroskar/workflow-run-cleanup-action@v0.3.0
       env:
         GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/master'"
+    if: "!startsWith(github.ref, 'refs/tags/') && !endsWith(github.ref, github.event.repository.default_branch)
```

```suggestion
    if: "!startsWith(github.ref, 'refs/tags/') && !endsWith(github.ref, github.event.repository.default_branch)"
```
I believe the `"` is missing...